02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,63 @@
+let { people } = require("../data");
+
+const getPeople = (req, res) => {
+  return res.json({ success: true, data: people });
+};
+
+const getPeopleId = (req, res) => {
```

Comment:
You could call this getPersonById, to make it more clearer that only one person is returned.